File: Tarea05/ejercicio4.py
conv = float(input("Ingresa 1 para convertir Celsius a Fahrenheit, 2 para convertir de Fahrenheit a Celcius\n"))
if conv == 1:
   grados = float(input("Ingresa los grados Celsius\n"))
   final = (1.8*grados) + 32
   print("%f grados Celsius son %f grados Fahrenheit"%(grados, final))
else:
   grados = float(input("ingresa los grados Fahreheit\n"))
   final = (grados - 32) / 1.8
   print("%f grados Fahrenheit son %f grados Celsius"%(grados, final))

File: Tarea05/ejercicio4.c
#include <stdio.h>

int main(){
	float grados, final;
	int conv;
	printf("Ingresa 1 para convertir Celsius a Fahrenheit, 2 para convertir de Fahrenheit a Celcius\n");
	scanf("%i", &conv);
	if (conv == 1){
		printf("Ingresa los grados Celsius\n");
		scanf("%f", &grados);
		final = (1.8*grados) + 32;
		printf("%f grados Celsius son %f grados Fahrenheit.\n", grados, final);
	}
	else{
		printf("Ingresa los grados Fahrenheit\n");
		scanf("%f", &grados);
		final = (grados - 32)/1.8;
		printf("%f grados Fahrenheit son %f grados Celsius\n", grados, final);
	}
	return 0;
}